### DISCUSSION TITLE
Integration of MDW Sky Survey

### DISCUSSION THREAD

**Discussion by GerritErdt** [Stellarium/stellarium#3560] [Ideas]
Hello, 

over the last years the MDW sky survey advanced, and during the last few days, they announced a first data release in January 2024!

The project can be found here: https://www.mdwskysurvey.org/

In summary, it aims at producing an all sky survey (similar to SDSS) consisting of deep, high quality, dark sky and high resolution H-Alpha images covering the north and south hemisphere. 

Since the original SDSS survey is quite old and of low quality, and most objects emit in the H-Alpha emission line, I think that the MDW sky survey would be a phenomenal tool for observation/photography planning and would make a great addition for Stellarium. 

Do you consider contacting the authors and including this survey in Stellarium?

Kind regards
Gerrit

**Comment by gzotti** [comment]
You could contact them. If they can provide their survey in HiPS format, you can simply include it.
